tests/conftest.py

```diff
@@ -365,7 +369,6 @@ def shutdown(self):
                 raise Exception("herbstluftwm did not quit on sigterm"
                                 + " and had to be killed") from None
 
-
```

Comment:
This was a load-bearing blank line, please keep intact :stuck_out_tongue:

Reply:
I don't get it, but I've added it again.

Reply:
Now I get it and moved the empty line to line 371